picoGPT/demo.py

```
@@ -0,0 +1,120 @@
+
+            issue_tags = cls.get_issue_tag(item.get("tag", ""), task)
+            res['issue_tags'] = issue_tags
+            res['mid'] = mid
+
+            resp_list.append(res)
+
+        # 问题数>1时，分别创建子记录task，分别处理
+        else:
+            for item in resp:
+                # 将review出问题的行和其上下三行更新到数据库以便之后提问题
+                new_task_dict = model_to_dict(task)
```

Ruff (0.8.2)

Undefined name `model_to_dict`

(F821)